downloads/index.html

```diff
@@ -110,23 +114,25 @@ <h2>Current stable release (v1.0.1)</h2>
         If the provided download files do not work for you, please <a href="https://github.com/JuliaLang/julia/issues">file an
         issue in the Julia project</a>.
       </p>
-
-      <p>
-        You may be interested, especially as an existing package author or maintainer, in downloading Julia v0.7.0, shown below.
```

Comment:
Why not just put this whole text block up earlier where you've added the smaller blurb?

Reply:
I thought that might be too long the have at the start.
What do you think?

Reply:
I think the information in it is important enough to warrant being there in its entirety. We can always remove it or pare it down once the 1.0 dust settles a bit.

Reply:
Ok